models/engine/db_storage.py

```diff
@@ -74,3 +74,18 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        all_objs = self.all(cls)
+        obj_id = cls.__name__ + '.' + id
+    
+        if obj_id in all_objs:
+            return all_objs[obj_id]
+        else:
+            return None
+    
+    def count(self, cls=None):
+        if cls != None:
+           return len(self.all(cls))
+        else:
+           return len(self.all())
```

Comment:
Hi Fabian, Don't forget to add comments for each function with description about its functionality and check the pep8 standard. @Fabian-Andres